google/cloud/spanner/row.h

```diff
@@ -167,6 +167,20 @@ class Row {
   }
   friend bool operator!=(Row const& a, Row const& b) { return !(a == b); }
 
+  /// Relational operators.
+  friend bool operator<(Row const& a, Row const& b) {
+    return a.values_ < b.values_;
+  }
+  friend bool operator<=(Row const& a, Row const& b) {
+    return std::rel_ops::operator<=(a.values_, b.values_);
```

Comment:
I don't think we gain much by using `std::rel_ops` here. I think it should be equivalent and perhaps a bit simpler to just have each of our relational operators forward to tuple's. So, how about instead just doing:

```cc
friend bool operator<=(Row const& a, Row const& b) {
  return a.values_ <= b.values_;
}
...
```
I think that'd be shorter too.

If you got this route, you probably want to fix the existing `operator!=` to follow this same pattern.